.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        php:
+          - '8.1'
+          - '8.2'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Lint PHP
+        run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s lint
+
+  code-quality:
+    name: Code Quality
+    runs-on: ubuntu-22.04
+    env:
```

Comment:
In this case the matrix should be used. The code snippet tool can run locally under 8.1, 8.2.